Poll drafts for march 31 (#2874)
makerdao/community: governance/polls/Spark Liquidity Layer Mainnet - Adjust USDC PSM Swap Rate Limits - March 31, 2025.md

---
title: Spark Liquidity Layer Mainnet - Adjust USDC PSM Swap Rate Limits - March 31, 2025

> summary: Signal your support or opposition to modifying the rate limits for Spark Liquidity Layer Mainnet USDS.

Reviewer: ```suggestion
summary: Signal your support or opposition to modifying the rate limits for Spark Liquidity Layer Mainnet USDC.
```

discussion_link: https://forum.sky.money/t/april-3-2025-proposed-changes-to-spark-for-upcoming-spell-2/26203
parameters:
    input_format: single-choice
    victory_conditions:
        - { type : plurality }
    result_display: single-vote-breakdown
version: v2.0.0
options:
   0: Abstain
   1: Yes
   2: No
start_date: 2025-03-31T16:00:00
end_date: 2025-04-03T16:00:00
---
# Poll: Spark Liquidity Layer Mainnet - Adjust USDC PSM Swap Rate Limits - March 31, 2025

> The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Stability Facilitators. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, March 31 at 16:00 UTC.

Reviewer 2: ```suggestion
The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Stability Facilitators. This Governance [Poll](https://sky-atlas.powerhouse.io/#A.1.9.1_Operational_Weekly_Cycle-b189fa17-57a9-4d4e-9780-0ce4efd94211%7C0db30308) will be active for three days beginning on Monday, March 31 at 16:00 UTC.
```

**This is a binary vote.**

- **You may vote for a single option.**
- **You should vote for the option which you prefer.**
- **If you would accept either option, you should vote 'Abstain'.**

## Review

> The community can vote in this poll to express support or opposition to the following rate limit parameters for Spark Liquidity Layer Mainnet USDC PSM Swap:

Reviewer: ```suggestion
The community can vote in this poll to express support or opposition to the following rate limit parameters for Spark Liquidity Layer Mainnet USDC PSM Swap.
```

Nit: We never do this when a header follows.
